test/e2e/framework/util.go

```diff
@@ -5192,18 +5192,25 @@ func WaitForPersistentVolumeClaimDeleted(c clientset.Interface, ns string, pvcNa
 	return fmt.Errorf("PersistentVolumeClaim %s is not removed from the system within %v", pvcName, timeout)
 }
 
-func GetClusterZones(c clientset.Interface) (sets.String, error) {
+func GetClusterZones(c clientset.Interface) (sets.String, error, error) {
+	notfoundPartNodes := []string{}
 	nodes, err := c.CoreV1().Nodes().List(metav1.ListOptions{})
 	if err != nil {
-		return nil, fmt.Errorf("Error getting nodes while attempting to list cluster zones: %v", err)
+		return nil, fmt.Errorf("Error getting nodes while attempting to list cluster zones: %v", err), nil
 	}
 
 	// collect values of zone label from all nodes
 	zones := sets.NewString()
 	for _, node := range nodes.Items {
 		if zone, found := node.Labels[kubeletapis.LabelZoneFailureDomain]; found {
```

Comment:
@verult In func getZoneNameForNode when zone is not found here, it will return error right now. In func GetClusterZones even PartNode zone not found, it will not return error. So I add an error for this case.